src/core/load-fragment.c

```diff
@@ -1277,6 +1278,15 @@ int config_parse_numa_mask(const char *unit,
                 return 0;
         }
 
+        c = container_of(p, ExecContext, numa_policy);
```

Comment:
Please don't do that. I'll fall apart badly when config_parse_numa_mask() is used in a different context. We already do that in `main.c`. Specifying NUMAMask= in system.conf will have funny results.